lib/tasks/step_nav/learn_to_drive_a_car.rake

```diff
@@ -0,0 +1,226 @@
+namespace :step_nav do
+  desc "Publish /learn-to-drive-a-car step navigation to publishing api"
+  task publish_learn_to_drive_a_car: :environment do
+    content_id = "e01e924b-9c7c-4c71-8241-66a575c2f61f"
+    params = {
+      base_path: "/learn-to-drive-a-car",
+      publishing_app: "collections-publisher",
+      rendering_app: "collections",
+      public_updated_at: Time.zone.now.iso8601,
+      update_type: "major",
+      change_note: "updating the content to match the new step_by_step_nav schema",
```

Comment:
@deborahchua I've just added this as I realised we were causing problems down the line by not providing a change note for a major update.

Reply:
Ah good point